src/components/app/matching/MatchesList.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { Match, MatchProps } from "./Match";
+
+export interface MatchesListProps {
+  matches: Array<MatchProps>
+}
+
+export const MatchesList = (props: MatchesListProps) => {
+
+  function renderMatch(match: MatchProps, listIdx: number) {
```

Comment:
we don't really need a named function here, we could rather use an anonymous function to define our matches like this:

```js
const matches = props.matches.map((match, id) => (<Match key={id} {...match} />))

return (
  <div>
    {matches}
  </div>
);
```

IMO, this is more readable than using a separate function and then mapping the list items inside the JSX below.